resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -6,10 +6,11 @@
     executor:
       docker:
         image: "resolwebio/archive-samples"
-  version: 0.0.2
+  version: 0.0.3
   type: "data:archive:samples"
   category: other
   persistence: TEMP
+  scheduling_class: interactive
```

Comment:
I think it would be naive to treat the archiver-like processes as interactive. Of course, if users "behave" and only download/create small archives, this will work, but in general, archives could be arbitrarily large.

The current implementation is actually vulnerable to filling-up the high priority Celery queue with archiver processes' tasks and all other (*really*) interactive processes will have to wait. Do we want that? I guess not.

Once we agree that archiver processes should not be treated as interactive, we can start discussing how to properly solve the archiver processes' use case.